satip/utils.py

```diff
@@ -301,6 +301,34 @@ def convert_scene_to_dataarray(
     return dataarray
 
 
+def do_old_rescaling(
```

Comment:
should we call this v0 rescaling? 

Just because we will continue to develop things and then it becomes confusing what is 'old'

Reply:
Yeah, good point, or I mnight go with v15 rescaling, as its then the same rescaling as the v15 dataset